script/load_templates

```diff
@@ -156,10 +156,14 @@ sub post_entry {
 
         # Post the job template YAML
         my $job_templates_url = $url->clone->path($options{'apibase'} . '/job_templates_scheduling');
-        $res
-          = $client->post($job_templates_url,
-            form => {name => $entry->{group_name}, template => $entry->{template}, schema => 'JobTemplates-01.yaml'})
-          ->res;
+        $res = $client->post(
+            $job_templates_url,
+            form => {
+                name     => $entry->{group_name},
+                foo      => 'bar',
+                template => $entry->{template},
+                schema   => 'JobTemplates-01.yaml'
+            })->res;
```

Comment:
Might make sense to use `->result` here. Then it would automatically throw an exception for connection errors.